### PR TITLE
Update office-365-management-activity-api-schema.md

### PR DESCRIPTION
**Pull request by pratjain523**
Updating the DLP schema to incorporate the new classification scheme results being pumped into audit logs.

### CODE REVIEW

File: office-365-management-api/office-365-management-activity-api-schema.md [hunk 1]
```diff
@@ -823,6 +823,7 @@ DLP (Data Loss Prevention) events will always have UserKey="DlpAgent" in the com
 
 |**Parameters**|**Type**|**Mandatory?**|**Description**|
 |:-----|:-----|:-----|:-----|
+|ConditionMatchedInNewScheme|Edm.Boolean|Yes|Indicates whether the classification results in new scheme (high, medium and low) result in DLP rule match.|
```
Comment (amaljg) [inline office-365-management-api/office-365-management-activity-api-schema.md:826]: Suggestion: Description can be `Indicates whether the new classification scheme (high, medium and low) results in DLP rule match or not.`

Comment (amaljg) [inline office-365-management-api/office-365-management-activity-api-schema.md:826]: Should we skip documenting `ConditionMatchedInNewScheme` altogether? It was actually meant for passive side-by-side mode scenarios. Once the new scheme is rolled out, this should always be true, right?

**Review by amaljg — APPROVED** [review]
Just double-check on exposing the `ConditionMatchedInNewScheme` property. Otherwise looks good.

**Comment by nivnar** [comment]
Tagging content manager of this repo @lindalu-MSFT to help publish this change. Thanks

FYI - @pratjain523, Please re-open this PR against `MicrosoftDocs:master` branch instead. After merging this, open a new PR from master to live branch for publishing the changes.

image

**Comment by pratjain523** [comment]
@Niveditha Narva<mailto:Niveditha.Narva@microsoft.com> Can you help in doing this? I do not have the expertise to do these. From my end, I have made the edits needed in content through github.

Thanks,
Prateek

**Pull request merged**